script.js

```diff
@@ -0,0 +1,338 @@
+  { code: '74', text: 'О', location: '0' }, { code: '75', text: 'Л', location: '0' }, { code: '76', text: 'Д', location: '0' },
+  { code: '186', text: 'Ж', location: '0' }, { code: '222', text: 'Э', location: '0' }, { code: '13', text: 'Enter', location: '0' },
+  { code: '16', text: 'Shift', location: '1' }, { code: '220', text: '\\', location: '0' }, { code: '90', text: 'Я', location: '0' },
+  { code: '88', text: 'Ч', location: '0' }, { code: '67', text: 'С', location: '0' }, { code: '86', text: 'М', location: '0' },
+  { code: '66', text: 'И', location: '0' }, { code: '78', text: 'Т', location: '0' }, { code: '77', text: 'Ь', location: '0' },
+  { code: '188', text: 'Ю', location: '0' }, { code: '190', text: 'Ю', location: '0' }, { code: '191', text: '/', location: '0' },
+  { code: '38', text: '▲', location: '0' }, { code: '16', text: 'Shift', location: '2' }, { code: '17', text: 'Ctrl', location: '1' },
+  { code: '91', text: 'Win', location: '0' }, { code: '18', text: 'Alt', location: '1' }, { code: '32', text: ' ', location: '0' },
+  { code: '18', text: 'Alt', location: '2' }, { code: '17', text: 'Ctrl', location: '2' }, { code: '37', text: '◄', location: '0' },
+  { code: '40', text: '▼', location: '0' }, { code: '39', text: '►', location: '0' }];
+
+const keyboard = document.createElement('div');
+keyboard.classList.add('keyboard');
+
+const wrapper = document.createElement('div');
+wrapper.classList.add('wrapper');
+
+const description = document.createElement('div');
+description.classList.add('description');
+description.innerHTML = '<div>Ctrl + Alt  - change language</div><div><i>Windows OS</i></div>';
+
+document.body.append(wrapper);
+
+class KeyBoard {
+  constructor() {
+    this.lang = JSON.parse(localStorage.getItem('myKeyboardLang')) || 'en';
+    this.activeKeyboard = enLowerCase;
+    this.shiftIsPressed = false;
+    this.altIsPressed = false;
+    this.ctrlIsPressed = false;
+    this.keyboard = document.createElement('div');
+  }
+
+  static createKeyElement({
+    code,
+    text,
+    location,
+  }) {
+    const key = document.createElement('div');
+    key.classList.add('key');
+    key.setAttribute('data-code', code);
+    key.setAttribute('data-location', location);
+    key.textContent = text;
+    return key;
+  }
+
+  static generateKeydownEvent(code) {
+    const event = new KeyboardEvent('keydown', {
+      bubbles: true,
+      cancelable: true,
+      keyCode: code,
+    });
+    document.dispatchEvent(event);
+  }
+
+  static highlightKey(key) {
+    key.classList.add('active');
+    setTimeout(() => {
+      key.classList.remove('active');
+    }, 300);
+  }
+
+  createKeyboard(btns) {
+    for (let i = 0; i < btns.length; i += 1) {
+      this.keyboard.append(KeyBoard.createKeyElement(btns[i]));
+    }
+
+    document.querySelectorAll('.key').forEach((item) => {
+      item.addEventListener('mousedown', () => {
```

Comment:
The same here about animation when I hold the button. Would be great to have a separate mouseup event listener to remove animation and 'active' class.